Uplift/Views/ClassDetail/ClassDetailHeaderView.swift

```diff
@@ -9,20 +9,26 @@
 import UIKit
 
 protocol ClassDetailHeaderViewDelegate: class {
+    func classDetailHeaderViewFavoriteButtonTapped()
+    func classDetailHeaderViewBackButtonPressed()
     func classDetailHeaderViewLocationSelected()
     func classDetailHeaderViewInstructorSelected()
+    func classDetailHeaderViewShareButtonTapped()
 }
 
 class ClassDetailHeaderView: UICollectionReusableView {
 
     // MARK: - Private view vars
+    private let backButton = UIButton()
     private let imageView = UIImageView()
     private let imageFilterView = UIView()
     private let semicircleImageView = UIImageView(image: UIImage(named: ImageNames.semicircle))
     private let titleLabel = UILabel()
     private let locationButton = UIButton()
     private let instructorButton = UIButton()
```

Comment:
Can you organize these based on type?

Reply:
These are also organized alphabetically so it's easy to identify views by their purported function, what would the benefits to organizing by type be?